src/main/java/com/commercetools/sync/products/helpers/ProductReferenceResolver.java

```diff
@@ -184,35 +202,30 @@ public ProductReferenceResolver(@Nonnull final ProductSyncOptions productSyncOpt
      * in the CTP project, the resultant draft would remain exactly the same as the passed product draft
      * (without reference resolution).
      *
-     * @param productDraft the product draft to resolve it's category references.
+     * @param draftBuilder the product draft builder where to update resolved category references.
```

Comment:
`the product draft builder where to update resolved category references.`
-> `draftBuilder the product draft builder to resolve it's category references.`

Reply:
51ead5e4